jenkins/scripts/bare_metal_lab/files/config.sh

```diff
@@ -1,168 +0,0 @@
-#
-#export FORCE_REPO_UPDATE="${FORCE_REPO_UPDATE:-false}"
-
-#
-# Run a local baremetal operator instead of deploying in Kubernetes
-#
-#export BMO_RUN_LOCAL=true
-
-#
-# Run a local CAPM3 operator instead of deploying in Kubernetes
-#
-#export CAPM3_RUN_LOCAL=true
-
-#
-# Do not retry on failure during verifications or tests of the environment
-# This should be true. It could only be set to false for verifications of a
-# dev env deployment that fully completed. Otherwise failures will appear as
-# resources are not ready.
-#
-#export SKIP_RETRIES=false
-
-#
-# Interval between retries after verification or test failure
-#
-#export TEST_TIME_INTERVAL=10
-
-#
-# Number of maximum verification or test retries
-#
-#export TEST_MAX_TIME=120
-
-#
-# Set the driver. The default value is 'mixed' (alternate nodes between ipmi
-# and redfish).
-#
-#export BMC_DRIVER="mixed"
-
-# Select the Cluster API provider Metal3 version
-# Accepted values : v1alpha3 v1alpha4
-# default: v1alpha4
-#
-
-#export KUBERNETES_VERSION="v1.20.2"
-
-# Version of kubelet, kubeadm and kubectl binaries
-#export KUBERNETES_BINARIES_VERSION="${KUBERNETES_BINARIES_VERSION:-${KUBERNETES_VERSION}}"
-#export KUBERNETES_BINARIES_CONFIG_VERSION="v0.2.7"
-
-# Configure provisioning network for single-stack ipv6
-#PROVISIONING_IPV6=false
-
-# Image OS (can be "Cirros", "Ubuntu", "Centos", overriden by IMAGE_* if set)
-#
-#export IMAGE_OS="Ubuntu"
-
-# Image for target hosts deployment
-#
-#export IMAGE_NAME="cirros-0.5.1-x86_64-disk.img"
-
-# Location of the image to download
-#
-#export IMAGE_LOCATION="http://download.cirros-cloud.net/0.5.1"
-
-# Image username for ssh
-#
-#export IMAGE_USERNAME="metal3"
-
-# Container image for ironic pod
-#
-#export IRONIC_IMAGE="fmuyassarov/ironic-image:latest"
-
-# Container image for vbmc container
-#
-#export VBMC_IMAGE="quay.io/metal3-io/vbmc"
-
-# Container image for sushy-tools container
-#
-#export SUSHY_TOOLS_IMAGE="quay.io/metal3-io/sushy-tools"
-
-# APIEndpoint IP for target cluster
-#export CLUSTER_APIENDPOINT_IP="192.168.111.249"
-
-# Cluster provisioning Interface
-#export CLUSTER_PROVISIONING_INTERFACE="ironicendpoint"
-
-# POD CIDR
-# export POD_CIDR=${POD_CIDR:-"192.168.0.0/18"
-
-# Node hostname format. This is a format string that must contain exactly one
-# %d format field that will be replaced with an integer representing the number
-# of the node.
-# export NODE_HOSTNAME_FORMAT="node-%d"
-
-# Ephemeral cluster used as management cluster for cluster API
-# (can be "kind", "minikube" or "tilt"). Only "minikube" is supported with
-# CentOS
-# Selecting "tilt" does not deploy a management cluster, it is left up to the
-# user
-
-# Secure Ironic deployment with TLS ("true" or "false")
-# export IRONIC_TLS_SETUP="true"
-
-# Set nodeDrainTimeout for controlplane and worker template, otherwise default value will be  '0s'. 
-#
-#export NODE_DRAIN_TIMEOUT=${NODE_DRAIN_TIMEOUT:-"0s"}
-
```

Comment:
I don't have a strong opinion on this, while this was used as a reference file to know what can be set or not, I am fine with removing it also, because the one in m3-dev-env keeps changing frequently and we always have to keep this file in sync with that.

Reply:
Yes I was thinking that it was confusing to have it here also, and since it is not really used other than for docs I think it makes sense to remove it from this repo and keep it in metal3-dev-env.

Reply:
+1